canary/package.json

```diff
@@ -8,9 +8,11 @@
     "pull": "./environments/pull-environments.sh",
     "script:install": "yarn --cwd script",
     "setup:next": "yarn script:install && yarn updatePackage:next",
+    "setup:reactv4": "yarn script:install && yarn updatePackage:reactv4",
     "setup:next-release": "yarn script:install && yarn updatePackage:next-release",
     "test": "yarn --cwd e2e test",
     "updatePackage:next": "node script/changeCanaryPackageVersion.mjs next",
+    "updatePackage:reactv4": "node script/changeCanaryPackageVersion.mjs react-v4-next",
```

Comment:
Do we need this if the _canaries_ directory is gone?

Reply:
I only should have removed `canary/apps/react/vite3`, but either way this code won't make it into our main branch :). It will only get pushed to `hotfix` and we will not be cherrypicking this to main because it wouldnt make sense to

Reply:
No worries, should have looked a bit closer. Thanks!